lua/telescope/utils.lua

```diff
@@ -392,4 +392,17 @@ utils.get_devicons = (function()
   end
 end)()
 
+function utils.create_tag_set(tag)
+  tag = vim.F.if_nil(tag, 'ordinal')
+  local set = {}
+  return setmetatable(set, {
+    __index = {
```

Comment:
Just thinking out load. There is also `__newindex` do we need to handle that as well?

Reply:
No, from reading documentation my understanding is that we don't need it here. Haven't used it and I quickly tried reeimplementing a set with `__newindex`. I might be clumsy but that use case is circular, since `__newindex` defines behavior for keys that are absent from the table. When trying to set a key for a table that's not there in `__newindex` it gets reinvoked. :sweat_smile: 

Sure thing :)